spinalcordtoolbox/download.py

```diff
@@ -310,7 +310,12 @@ def install_data(url, dest_folder, keep=False):
             for entry in it:
                 if entry.name in ("__MACOSX",):
                     continue
-                bundle_folder = entry.path
+                # NB: Don't traverse into nnUNetTrainer folders, and instead preserve the structure.
+                # This allows users to re-use the model folder for 3DSlicer with the correct structure.
+                elif entry.name.startswith("nnUNetTrainer"):
+                    bundle_folder = extraction_folder
+                else:
+                    bundle_folder = entry.path
```

Comment:
I don't love having a reference to `nnUNetTrainer` inside of `download.py`, which is more "general"... :thinking:

Reply:
Done in https://github.com/spinalcordtoolbox/spinalcordtoolbox/pull/4583/commits/913401180c9f6210df0cdaba73a33d423b27cec7.